### PR TITLE
SPUI-909 > Fix margins in market banners

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/app/screens/market/view/components/market_banners/market_banners.dart [hunk 1]
```diff
@@ -27,30 +27,33 @@ class MarketBanners extends HookWidget {
...
+                    index: index,
+                    bannersLength: state.length,
+                    color: randomBannerColor(colors),
+                    primaryText: campaign.title,
+                    imageUrl: campaign.imageUrl,
+                    primaryTextStyle: sTextH5Style,
+                    onClose: () {
+                      notifier.deleteCampaign(campaign);
+                    },
+                    // indentRight: state.length == 1,
```
Comment (eli1stark) [inline lib/app/screens/market/view/components/market_banners/market_banners.dart:52]: remove comment

File: plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    this.secondaryText,
+    this.imageUrl,
+    this.onClose,
+    this.primaryTextStyle,
+    this.indentRight = false,
+    required this.color,
+    required this.primaryText,
+    required this.index,
+    required this.bannersLength,
+  }) : super(key: key);
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart:19]: add space here

File: plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  final String primaryText;
+  final bool indentRight;
+  final int index;
+  final int bannersLength;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: EdgeInsets.only(
+        right: (bannersLength != 1 && bannersLength == index + 1)
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart:34]: add explanation comment what it does and why it's here

File: plugins/simple_kit/lib/src/banners/rewards_banner/simple_rewards_banner.dart [hunk 4]
```diff
@@ -9,97 +9,86 @@ class SRewardBanner extends StatelessWidget {
...
-  final Function()? onClose;
+
   final String? secondaryText;
   final String? imageUrl;
   final TextStyle? primaryTextStyle;
   final Color color;
   final String primaryText;
   final bool indentRight;
+  final int? index;
+  final int? bannersLength;
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/banners/rewards_banner/simple_rewards_banner.dart:25]: sort  parameters

File: plugins/simple_kit/lib/example/banners/rewards_banner/simple_rewards_banner_example.dart [hunk 5]
```diff
@@ -21,7 +21,6 @@ class SimpleRewardsBannerExample extends StatelessWidget {
               color: SColorsLight().violet,
               primaryText: 'Invite friends\nand get \$10',
               secondaryText: 'Get a rando m coin with every trade over \$50',
```
Comment (eli1stark) [inline plugins/simple_kit/lib/example/banners/rewards_banner/simple_rewards_banner_example.dart:23]: fix typo, rando m to random

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/screens/market/view/components/market_banners/market_banners.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/app/screens/market/view/components/market_banners/market_banners.dart:52]: done

File: plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart:19]: done

File: plugins/simple_kit/lib/example/banners/rewards_banner/simple_rewards_banner_example.dart (hunk 5 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/example/banners/rewards_banner/simple_rewards_banner_example.dart:23]: done

File: plugins/simple_kit/lib/src/banners/rewards_banner/simple_rewards_banner.dart (hunk 4 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/banners/rewards_banner/simple_rewards_banner.dart:25]: remove params

File: plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart (hunk 3 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/banners/market_banner/simple_market_banner.dart:34]: done

**Pull request merged**